lib/character.rb

```diff
@@ -0,0 +1,10 @@
+class Character
+attr_reader :name, :actor, :salary
+
+  def initialize(info)
+    @name = info.fetch(:name)
+    @actor = info.fetch(:actor)
+    @salary = info.fetch(:salary)
+  end
+
+end
```

Comment:
good work in here, don't forget to indent your attr_reader lines in these classes as well